graphbolt/src/concurrent_id_hash_map.cc

```diff
@@ -65,10 +65,10 @@ torch::Tensor ConcurrentIdHashMap<IdType>::Init(
   // This code block is to fill the ids into hash_map_.
   auto unique_ids = torch::empty_like(ids);
   IdType* unique_ids_data = unique_ids.data_ptr<IdType>();
-  // Fill in the first `num_seeds` ids.
-  torch::parallel_for(0, num_seeds, kGrainSize, [&](int64_t s, int64_t e) {
+  // Insert all ids into the hash map.
+  torch::parallel_for(0, num_ids, kGrainSize, [&](int64_t s, int64_t e) {
     for (int64_t i = s; i < e; i++) {
-      InsertAndSet(ids_data[i], static_cast<IdType>(i));
+      InsertAndSetSmaller(ids_data[i], static_cast<IdType>(i));
```

Comment:
A little confused with the name, could you change to a more meaningful name?

Reply:
How about `InsertAndSetIfSmaller` or `SetIfSmaller`

Reply:
InsertAndSetMin is clearer to me, this loop is finding the minimum value, right?
```
  while (old_val == empty_key || old_val > value) {
    old_val = CompareAndSwap(&(hash_map_data[val_pos]), old_val, value);
  }
```